kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -113,11 +113,22 @@ class ResidualBasedBlockBuilderAndSolver
     ///@name Life Cycle
     ///@{
 
-    /** Constructor.
+    /**
+     * @brief Default constructor. (with parameters)
+     */
+    explicit ResidualBasedBlockBuilderAndSolver(
+        typename TLinearSolver::Pointer pNewLinearSystemSolver,
+        Parameters ThisParameters
+        ) : BaseType(pNewLinearSystemSolver, ThisParameters)
+    {

```

Comment:
no validation?

Reply:
I will not force the validation as the parameters that can be passed can contain many things, personally I am against

Reply:
For example @RiccardoRossi look at the constructor, calls the base class, which could have a different validator, then throwing an error. That's why there is no validation

Reply:
The validator of this class wins over the one of the base class. If the parameters are different then you will have to pass proper parameters to the base class

i consider this as very important

Reply:
If parameters are not compatible, you will have to construct the base class using a constructor that does not pass parameters

Reply:
OK